src/content/1.7/modules/concepts/templating/vuejs/faq.md

```diff
@@ -27,6 +27,77 @@ Example:
     ]);
 ```
 
+### How to send data from Symfony modern controllers to Vue.js?
+
+PHP controller:
+
+```php
+namespace Moduel\YorModule\Controller\Admin;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+
+class YourControllerNameController extends FrameworkBundleAdminController
+{
+    public function demoAction()
+    {
+        return $this->render('@Modules/your-module/views/templates/admin/demo.html.twig', [
+            'yourModule' => [
+                'keyA' => 'valueA',
+                'keyB' => 'valueB',
+            ]
+        ]);
+    }
+}
+```
+
+Twig template:
+
+```twig
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Module.YourModule.Admin.YourControllerName" %}
```

Comment:
From what I remember we can't use more than 2 levels of domain in custom translation domains?

Can you confirm @eternoendless @sowbiba ?